dependencies.yml

```diff
@@ -57,6 +63,12 @@ io.zipkin.brave:
 it.unimi.dsi:
   fastutil: { version: 7.2.0 }
 
+javax.inject:
+  javax.inject: { version: !!str 1 }
```

Comment:
Just wondering, does `'1'`, string syntax, work too? If so it's a bit simpler than this weird syntax